rio/ChangeLog.md

```diff
@@ -15,6 +19,7 @@
 * Re-export `zipWith` and `runST` from `RIO.Prelude`
 * Re-export `Exception`, `MonadFail`, `Typeable` and `ST` from `RIO.Prelude.Types`
 * Switch to `MonadFail.fail` from `Monad.fail` and re-exported it from `RIO.Prelude`
+* Functions from `Data.Data.Data` class are no longer exported.
```

Comment:
Is this an intentional change versus previous releases?

Reply:
It was not intentional, but I totally support it. Generic programming has no place in Prelude IMHO. So it might have been a subconscious decision ;) Just kidding of course. I was going type, by type, class by class, and was trying to make the previous PR having least semantic changes possible, but some things slipped through.

Reply:
The type class itself is exported, which is very useful for deriving.

Reply:
I think I'd rather add them back. They haven't caused any trouble that I'm aware of, and I'd rather avoid breaking backwards compatibility if we can.

Reply:
I'll humbly disagree. But it's your call, so I added them back to the reexport list